### DISCUSSION TITLE
webrtc [RTSP source] dial tcp xxx.xxx.xxx.xxx:xxx connect: connection refused

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2504] [Questions]
### Question

[RTSP source] dial tcp xxx.xxx.xxx.xxx:xxx connect: connection refused
I am getting this error can you please help me.

**Comment by aler9** [comment]
Hello, this error means that your PC is unable to reach the camera. Make sure that the camera is connected and that its stream can be pulled with RTSP. Try using the RTSP address of the camera inside VLC to be sure.

[a comment by an automated account is not shown]
